lib/govuk_publishing_components/app_helpers/countdown_helper.rb

```diff
@@ -0,0 +1,33 @@
+module GovukPublishingComponents
+  module AppHelpers
+    class CountdownHelper
+      END_OF_TRANSITION_PERIOD = Date.new(2021, 1, 1)
+
+      def days_left
+        sprintf "%02d", days_left_integer
+      end
+
+      def show?
+        days_left_integer.positive?
+      end
+
+      def days_text
+        if days_left_integer == 1
+          I18n.t("components.transition_countdown.day_to_go")
```

Comment:
You might want to use `I18n.t!` here so that an exception is raised if the translation doesn't exist (otherwise it'll display `components.transition_countdown.day_to_go` which can be hard to catch)

Reply:
Updated with your suggestion for a more robust approach